packages/suite-web-landing/src/static/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en" translate="no">
+    <head>
```

Comment:
please compare the head section on this branch with the current one (and this file with original deleted _document.tsx)

Reply:
for example onion-location and cache directives are missing

Reply:
14a42c2